godtools/Views/TractElements/TractCardProperties.swift

```diff
@@ -61,5 +61,6 @@ class TractCardProperties: TractProperties {
     
     var cardState = CardState.preview
     var cardNumber = 0
+    var cardIdName = ""
```

Comment:
This name is a little bit confusing to me. I'm not sure at face value what value it represents.

Reply:
I've changed this to cardLetterName.  I think that communicates what we're trying to get at now